recipe/meta.yaml

```diff
@@ -1,21 +1,22 @@
-{% set version = "16.0.0" %}
+{% set version = "16.2.0" %}
 
 package:
   name: path
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/p/path/path-{{ version }}.tar.gz
-  sha256: 48ddde18c6a15d40233bffeca882b6297a3e3a058903f1d4b52671cb8efd4018
+  sha256: 2de925e8d421f93bcea80d511b81accfb6a7e6b249afa4a5559557b0cf817097
 
 build:
   number: 0
   noarch: python
   script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [py<36]
```

Comment:
`skip: true` using Python version selectors should not be required or used for `noarch: python` packages.

Reply:
the skip section was removed